README.md

```diff
@@ -0,0 +1,95 @@
+# FarmLink
+Our capstone project is a web application that helps FarmLink, a non-profit organization that moves food surplus from farms around the country to food banks in need, automate the process of matching a food bank with a farm.
+# How to get application running!
+In this README file we will briefly explain how to get this application up and running on your computer. Feel free to reach out to the contributors to the repo for any further assistance.
+## Step #1: Clone the Repository
+Navigate to the directory where you would like to clone the repository. Then navigate to the main page of the repository, press the green code button, and then copy the link under "Clone with HTTPS".
+
+Now enter the following command into terminal:
+
+    git clone https://hostname/YOUR_USERNAME/YOUR-REPOSITORY
+At this point all the folders within the repository should have successfully downloaded locally.
+## Step #2: Download Dependencies 
+If you haven't installed npm, navigate to this [download tutorial](https://www.npmjs.com/get-npm) and do so before continuing.
+
+Now navigate to the view directory within the code base an run the following command:
+
+    ~/farmlink/view npm install
+
+Now repeat the process in the functions directory:
+
+    ~/farmlink/functions npm install
+Now you can test if the React application runs by running the following command in the view directory (if it doesn't run, you will likely need to download a missing dependency)
+
+    ~/farmlink/functions npm start
+If it runs, then you are good to go onto the next step!
+## Step #3: Setup Firebase
+Create a project in Firebase and run the following after doing so:
+
+    ~/farmlink/functions npm install -g firebase-tools
+    firebase init
+Once you have run `firebase init`, answer the prompts as follows:
+
+    Which Firebase CLI features do you want to set up for this folder? 
+    => Functions: Configure and deploy Cloud Functions
+    First, let’s associate this project directory with a Firebase project...
+    => Use an existing project
+    Select a default Firebase project for this directory
+    => application_name 
+    Do you want to use ESLint to catch probable bugs and enforce style?
+    => N
+    Do you want to install dependencies with npm now?
+    => Y
+After this, you should see the following message:
+
+    ✔ Firebase initialization complete!
+## Step #3: Setup Firebase Firestore
+Go to the database tab in Firebase console and create a database set to `test mode` and choose `us-central` as your hosting location.
```

Comment:
Does it really matter what they choose as their hosting location?  I would expect they'd want to pick the one closest to their users.